### ISSUE TITLE
File import

### ISSUE THREAD

**Issue by gshotwell** [whitphx/quarto-stlite#1]
One of the common things that I do with shinylive is to reference files in code. In shinylive this is done by adding a `## file: <file_name>` heading after the main app script. I'm not sure how this would integrate with stlite file serving, but it would be good if there were a way to add other files from the quarto interface.

**Comment by whitphx** [comment]
Makes sense, and inputting multiple files to the stlite runtime is already possible.

Let me ask about the question on the custom parser for it.
Looks like Shinylive has implemented its custom parser to deal with the special syntax such as `## file:` or `#| `.
Do you think it also would be the best practice for the `quarto-stlite` extension, or are there some reusable library to handle these things?

**Comment by gshotwell** [comment]
There's not (yet) a reusable file import pattern for quarto chunks like this which is too bad, it might be worth raising an issue on the quarto repository to ask for one since this might keep coming up.

**Comment by whitphx** [comment]
Created a discussion thread: https://github.com/quarto-dev/quarto-cli/discussions/6431
